src/pages/order-item/tabs/tests/add-product-form-tests/add-product-form.variables.js

```diff
@@ -1,3 +1,7 @@
+const {
+  getPromoCodeByCode
+} = require('../../../../promo-code/operations/promo-code.queries');
```

Comment:
I think you should not use CommonJS import on the frontend

Reply:
changed